drivers/i2c/i2c_dw.c

```diff
@@ -738,16 +753,15 @@ static int i2c_dw_transfer(const struct device *dev, struct i2c_msg *msgs, uint8
 
 static int i2c_dw_runtime_configure(const struct device *dev, uint32_t config)
 {
-	struct i2c_dw_dev_config *const dw = dev->data;
-	uint32_t value = 0U;
-	uint32_t rc = 0U;
+	struct i2c_dw_dev_config * const dw = dev->data;
+	uint32_t	value = 0U;
+	int		ret = 0;
```

Comment:
We usually do not indent variable names like this in Zephyr. Please leave this change out of this PR.